### ISSUE TITLE
Stripping spaces from card numbers

### ISSUE THREAD

**Issue by lukefrake** [stripe/jquery.payment#167]
When typing 4 digits payment.js would add a space between the fourth and fifth.

When the 5th digit is typed, the space is then stripped out.

This can be seen in [the demo](http://stripe.github.io/jquery.payment/example/) but isn't seen in @maccman's blog post about [payment.js](https://stripe.com/blog/jquery-payment) so I assume this isn't supposed to happen?

Am I missing something?

**Comment by lukefrake** [comment]
I have rolled back to version 1.1.4 which seems to not have this bug in. 

FYI @jamesreggio

**Comment by lukefrake** [comment]
Further digging, I think this started on [this commit](https://github.com/stripe/jquery.payment/commit/3dc241e9188407042026c6d39ee74480504c3715) by @jamesreggio 

Still unsure if this is a feature or a bug though, so will wait on clarification but then I don't mind making a pull request if it's a bug.

**Comment by luongvanphuc** [comment]
Is that also my concern at https://github.com/stripe/jquery.payment/issues/163 ? If yes, I've been waiting for a fix.

**Comment by tomsommer** [comment]
It's working as intended, according to #163

**Comment by HashNuke** [comment]
@lukefrake IMHO there's no proper way to space-out credit card numbers (so spacing out invalid ones is another problem). The length of card numbers can be anywhere from 12 to 19. https://en.wikipedia.org/wiki/Bank_card_number

**Comment by lukefrake** [comment]
Sorry @luongvanphuc I didn't see your second issue. Yeah thats exactly my problem!

I'll close this issue as it's a duplicate of yours.
